common/utils/request.js

```diff
@@ -54,3 +54,61 @@ export const post = async ({ hostname = 'hcm.show', path, body }) => {
     console.error(`Fetch error: ${JSON.stringify(err, null, 2)}`);
   }
 };
+export const getEmployees = async ({ hostname = 'hcm.show', path, params }) => {
```

Comment:
A better name for this function would be `get`, there's nothing anything employee specific here